docs/contracts/node-operators-registry.md

```diff
@@ -3,7 +3,32 @@
 - [Source Code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/nos/NodeOperatorsRegistry.sol)
 - [Deployed Contract](https://etherscan.io/address/0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5)
 
-Node Operators act as validators on the Beacon chain for the benefit of the protocol. The DAO selects node operators and adds their addresses to the NodeOperatorsRegistry contract. Authorized operators have to generate a set of keys for the validation and also provide them with the smart contract. As Ether is received from users, it is distributed in chunks of 32 Ether between all active Node Operators. The contract contains a list of operators, their keys, and the logic for distributing rewards between them. The DAO can deactivate misbehaving operators.
+The `NodeOperatorsRegistry` contract acts as a registry of Node Operators selected by Lido DAO.
+After [Lido V2 upgrade](https://blog.lido.fi/introducing-lido-v2/) `NodeOperatorsRegistry` contract became a module of [`StakingRouter`](./staking-router.md). The new name for `NodeOperatorsRegistry` contract as part of the entire Lido staking platform is **Curated staking module**. As a staking module, `NodeOperatorsRegistry` supports [StakingModule interface](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/interfaces/IStakingModule.sol).
+
+As Ether is received from [`StakingRouter`](./staking-router.md) on oracle report, it is distributed in chunks of 32 Ether between all active, not penalized Node Operators.
```

Comment:
I feel like this sentence is a bit surprising in this context, and don't follow its meaning: why we connect the oracle report **AND** 32 ETH?

Reply:
I believe, it's a mistake. SR receives ether during deposit flow, not on rebase. 

And, please, lets decapitalize the `Ether`, it's not used like that on ethereum.org and I don't see any reason to continue that trend from the early days docs.